distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
```

Comment:
I could do that but then the module name wouldn't match and it would be a convention violation to allow it to in that case; technically I'm already breaking convention adding a dash before `network` in this case but it helps readability. 

If docs don't have to match the original maven module then I'm ok with this.